exercises/perfect-numbers/description.md

```diff
@@ -1,11 +1,10 @@
 # Description
 
-Determine if a number is perfect, abundant, or deficient based on
-Nicomachus' (60 - 120 CE) classification scheme for positive integers.
+Determine if a number is perfect, abundant, or deficient based on Nicomachus' (60 - 120 CE) classification scheme for positive integers.
 
 The Greek mathematician [Nicomachus][nicomachus] devised a classification scheme for positive integers, identifying each as belonging uniquely to the categories of **perfect**, **abundant**, or **deficient** based on their [aliquot sum][aliquot-sum].
 The aliquot sum is defined as the sum of the factors of a number not including the number itself.
-For example, the aliquot sum of 15 is (1 + 3 + 5) = 9
+For example, the aliquot sum of 15 is: (1 + 3 + 5) = 9
```

Comment:
I don't think the colon is needed/correct here.

Reply:
Anything that helps delineate code/math from the surrounding English is welcome IMHO. @kotp 's reasoning :

> My reasoning for the change is that adding a period to the end of the sentence is confusing, potentially, in terms of programming. A colon to indicate the statement is not done, providing a statement to the right may provide clarity (and avoid the period).

https://github.com/exercism/raku/pull/669#discussion_r1316213674

Reply:
You could delineate the math with backticks or put it in a codeblock on a separate line. So long as it is in line in a sentence, it should follow grammatical rules; if you're making changes which don't follow rules, you're just setting it up for a follow up PR which will fix the prose to be correct grammar.

Reply:
Happy to use backticks. We call them rules, but I feel grammar is somewhat flexible and accommodating to evolving needs, particularly at the confluence/boundary of multiple languages. For example, I wrote above "@kotp 's" with a space so as to not create confusion with a user named "@kotp's", yet that isn't syntactically sound either.

Reply:
I'm a bit confused.

> Anything that helps delineate code/math from the surrounding English is welcome IMHO.

Is this a common preference? I have never encountered it before, but I might be living in an weird bubble.

Also: why the parentheses? More often I encounter 1 + 3 + 5 = 9, or rather $1 + 3 + 5 = 9$.

And: if the calculation is to be surrounded with backticks, then why not 15 as well? After all, they are of the same 'type': mathy squiggles denoting an integer.

Reply:
> Is this a common preference, and is there a reason for it? I had never encountered it before, but I might be living in an weird bubble.

It can make things easier to read when different "contexts" are visually different.

> Also: why the parentheses? More often I encounter 1 + 3 + 5 = 9, or rather 1+3+5=9.

I think the math-quoting is not supported by Exercism's MD parser.

> And: if the calculation is to be surrounded with backticks, then why not 15 as well? After all, they are of the same 'type': mathy squiggles denoting an integer.

I think that was an oversight.

Reply:
Yes, thanks !

Another PR :
https://github.com/exercism/problem-specifications/pull/2319